server/lib/gateway/gateway.downloadBackup.js

```diff
@@ -20,15 +21,15 @@ async function downloadBackup(fileUrl) {
   if (encryptKey === null) {
     throw new NotFoundError('GLADYS_GATEWAY_BACKUP_KEY_NOT_FOUND');
   }
-  // Extract file name
-  const fileWithoutSignedParams = fileUrl.split('?')[0];
+
+  const value = isURL(fileUrl) ? validateUrl(fileUrl) : fileUrl;
```

Comment:
I'm not sure it still works like that, for context, the fileUrl looks like that (it's a dummy URL) : 

```
https://example_storage.com/b6131425-8f90-4a05-ba00-12daf37f8279.enc?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=SDFDSFDSFDSFDSFFDF%2F20241202%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20241202T110928Z&X-Amz-Expires=21600&X-Amz-Signature=6768767678HHkjhjdhfjdshfjhdksfhjdshf&X-Amz-SignedHeaders=host
```

So the get parameters should be removed before doing path.basename()

Having the variable being name `value` isn't very clear